<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	border: opx;
	background-color: rgb(235, 235, 235);
}

QGroupBox#MenuGroupBox QPushButton:hover {
	background-color: rgb(212, 212, 212);
}
QGroupBox#MenuGroupBox QPushButton:focus {
	background-color: rgb(190, 190, 190);
}

QGroupBox#ProfileGroupBox QLineEdit {
	border: 1px solid gray;
}
QGroupBox#ProfileGroupBox QLabel#PhotoIcon {
	border: 1px solid gray;
}

QPushButton#exitButton:hover {
	background-color: rgb(255, 81, 46);
}
QPushButton#scrollDownButton:hover {
	background-color: rgb(174, 255, 250);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="titleGroupBox">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>36</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="IconLabel">
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>imgs/titleIcon.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NameLabel">
         <property name="text">
          <string>NetInspector</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scrollDownButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>—</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="MainGroupBox">
      <property name="minimumSize">
       <size>
        <width>639</width>
        <height>341</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="MenuGroupBox">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ProfileOpenButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Профиль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ProfileredactorOpenButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Управление пользователями</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="VoidGroupBox">
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExitProfileButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Выход</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ProfileGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QGroupBox" name="UserGroupBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>540</width>
            <height>364</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="PhotoIcon">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>imgs/png-transparent-avatar-business-.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="StatusLabelText">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>10</y>
             <width>50</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Статус:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="LoginLabel">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>35</y>
             <width>50</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Логин:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="PasswordLabelText">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>60</y>
             <width>50</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Пароль:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="StatusLabel">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>120</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Админ</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>35</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>60</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="LoginRedactorButton">
           <property name="geometry">
            <rect>
             <x>435</x>
             <y>35</y>
             <width>20</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/redactoricon_1.png</normaloff>imgs/redactoricon_1.png</iconset>
           </property>
          </widget>
          <widget class="QPushButton" name="PasswordRedactorButton">
           <property name="geometry">
            <rect>
             <x>435</x>
             <y>60</y>
             <width>20</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/redactoricon_1.png</normaloff>imgs/redactoricon_1.png</iconset>
           </property>
          </widget>
          <widget class="QPushButton" name="SaveNewLoginButton">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>35</y>
             <width>20</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/saveicon_1.png</normaloff>imgs/saveicon_1.png</iconset>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>85</y>
             <width>20</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/watch_pass.png</normaloff>imgs/watch_pass.png</iconset>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>60</y>
             <width>20</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/saveicon_1.png</normaloff>imgs/saveicon_1.png</iconset>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
